code/modules/mob/living/carbon/human/ai/brain/ai_brain_targeting.dm

```diff
@@ -34,7 +34,7 @@
 		if(!can_target(potential_target))
 			continue
 
-		if(!(potential_target in viewers(tied_human))) // for now, only consider targets in view
+		if(!(tied_human in viewers(potential_target))) // for now, only consider targets in view
```

Comment:
why did you reverse this

Reply:
Basically, what we were doing before is checking whenever our target sees us to determine enviroment environment opacity, which led to some funny business considering we are in viewers() even when using thermal glasses. Now it's reversed, and AI can only see people through walls if it has thermal/xray glasses